### ISSUE TITLE
Use classification visualizers directly from predictions, targets and logits?

### ISSUE THREAD

**Issue by 2533245542** [DistrictDataLabs/yellowbrick#1289]
Hi,

I work on classification problems and really like the design of the classification visualizers and their plots.

Nevertheless, I am a pytorch user. I usually store the model's output on test set as "prediction", "target", and "logits" (probability of each class).

It looks to me that classification report, confusion matrix, ROCAUC, precision-recall curves, class prediction error, discrimination threshold can be achieved using these three inputs.

Is there an easy way to adapt my workflow to it?

Thanks

**Comment by rebeccabilbro** [comment]
Hello @2533245542 and thank you for reaching out. 

Yellowbrick is designed to work with the scikit-learn API, meaning our visualizers expect models to have either `fit()` and `predict()` or `fit()` and `transform()`. If the third party library you are using does not implement that interface, many Yellowbrick users have had some luck writing a [custom wrapper](https://www.scikit-yb.org/en/latest/api/contrib/wrapper.html#tested-libraries). The documentation about all of this can be found [here](https://www.scikit-yb.org/en/latest/api/contrib/wrapper.html).

Let us know how it goes!
